Gordon360/Controllers/AccountsController.cs

```diff
@@ -118,6 +121,9 @@ public async Task<ActionResult<IEnumerable<BasicInfoViewModel>>> SearchWithSpace
             string? minor,
             string? hall,
             string? classType,
+            string? preferredClassYear,
```

Comment:
Would it make sense for `preferredClassYear` to be an int as well?

Reply:
It is not necessary because since we are only trying to find the exact year that matches the input as:
![image](https://github.com/gordon-cs/gordon-360-api/assets/115328139/2665f13f-9168-45e2-84b1-0d01cf345ba4)
it works fine beacus both preferredClassYear and a.PreferredClassYear are strings